### PR TITLE
Documentation v2 :memo: 

### PR DESCRIPTION
**Pull request by sreichel**
### Description (*)

Use `mkdocs` for better documentaion at https://openmage.readthedocs.io

### Related Pull Requests

1. See OpenMage/magento-lts#2984

### Manual testing scenarios (*)

1. follow https://github.com/OpenMage/magento-lts/commit/74507da7d57fcfdc26b8870f551ee47124ed3c0b to install `mkdocs`
2. run `mkdocs serve`

### Questions or comments

__Pros:__

- docs not splited to readme, readthedocs, docs direcory
- modern _flare_
- easier to extend (!)
- should work with readthdocs.io
- ...

__Cons:__

Have to install python3 (?)

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All automated tests passed successfully (all builds are green)
 - [x] Add yourself to contributors list

### CODE REVIEW

File: docs/users/modules/more.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Some more Modules, which we did not put into a an own category yet,
+some are also just placeholders waiting to be filled with module suggestions.
+
+## Cron Scheduling
+
+- https://github.com/AOEpeople/Aoe_Scheduler
+
+## Gift Cards
+
+- https://unirgy.com/products/ugiftcert (commercial)
```
Comment (rnicklin) [inline docs/users/modules/more.md:12]: Unirgy no longer offers this module for M1. Probably not worth including in this documentation.

Comment (m-overlund) [reply docs/users/modules/more.md:12]: are you sure? 
we have it, and they released a new version in april this year - they still develop it.

**Comment by sreichel** [comment]
> Need some help to automate pushing to [openmage.readthedocs.com](https://openmage.readthedocs.io/en/latest/).

- https://github.com/marketplace/actions/deploy-mkdocs
- https://squidfunk.github.io/mkdocs-material/publishing-your-site/#with-github-actions-material-for-mkdocs

**Comment by Flyingmana** [comment]
ReadTheDocks works pull not push based.

I can change repository and documentation type after the merge, but there is no way to build the documentation ourself there.

image


I did not follow the discussion around the documentation to the end, can you link to the decision in your PR for a cross check?

**Comment by sreichel** [comment]
Seems mkdocs-support is built in ... https://docs.readthedocs.io/en/stable/intro/getting-started-with-mkdocs.html

(point docs to RTD should work out of th box)

**Comment by sreichel** [comment]
> can you link to the decision in your PR for a cross check?

Decision about Wiki/Pages/Else discussion? Guess it was in the githubio-repo ?!?  Have to search for it.

**Pull request closed without merging**